### PR TITLE
Improving the exception messages for the FixtureManager.

### PR DESCRIPTION
**Pull request by burzum**
Before the message was very generic and not telling you which fixture caused the failure.

This has been pretty annoying because you could only guess which fixture caused the problem. The new messages will make that clear. IMO an exception message should be helpful and save me time and work by being clear and not just telling me "Oh X went wrong, but I'm not telling you where and what!"

### CODE REVIEW

File: src/TestSuite/Fixture/FixtureManager.php [hunk 1]
```diff
@@ -287,15 +292,25 @@ public function load($test)
                 }
 
                 foreach ($fixtures as $name => $fixture) {
-                    $fixture->createConstraints($db);
+                    try {
+                        $fixture->createConstraints($db);
+                    } catch (PDOException $e) {
+                        $msg = sprintf('Unable to create constriants for fixture "%s" for "%s" test case: ' . "\n" . '%s', get_class($fixture), get_class($test), $e->getMessage());
```
Comment (dereuromark) [inline src/TestSuite/Fixture/FixtureManager.php]: typo constraints
and maybe `for fixture "%s" in "%s" test case`

**Comment by dereuromark** [comment]
Is it intentional to change the exception from PDOException to Exception?
Would there be any downside of keeping the same type (rethrowing)?

**Comment by burzum** [comment]
@dereuromark that's what the existing, but unclear, exception at the end of the message did. I just followed the existing code.

File: src/TestSuite/Fixture/FixtureManager.php (hunk 1 above)
Comment (burzum) [inline src/TestSuite/Fixture/FixtureManager.php]: Fixed.

**Comment by markstory** [comment]
Nice work @burzum

**Pull request merged**